### ISSUE TITLE
Incompatible with ui-router 1.0+

### ISSUE THREAD

**Issue by sjmorrow** [vinaygopinath/ngMeta#25]
ui-router has deprecated the use of `$stateChangeSuccess` in 1.0, which completely breaks `ngMeta.init()`.

Relevant documentation on the new ui-router site: https://ui-router.github.io/guide/ng1/migrate-to-1_0#state-change-events

**Comment by mackelito** [comment]
Damn.. I´m just about to get started on adding this module to our app.. to bad it doesn't´t work whit 1.0

I guess that this should be easy to update and just use the `$transitions.onSuccess` instead.

**Comment by mackelito** [comment]
From the ui-router repo.

The $stateChange* events are deprecated in 1.0. They are replaced with the $transitions.on* transition hooks.

However, they can be re-enabled for backwards compatibility by including stateEvents.js and depending on the 'ui.router.state.events' angular module.

```javascript
<script src="stateEvents.js">
var myApp = angular.module('myApp', ['ui.router', 'ui.router.state.events']);
```

Note: the code in stateEvents.js re-implements the $stateChange* events using the new Transition Hooks

**Comment by taasver** [comment]
I didn't want to re-enable state change events, so I did this HACKY solution that works on my case:
```
// file: app.run
$transitions.onFinish({}, function(trans) {
  $rootScope.$broadcast('$routeChangeSuccess', trans.to());
});
```
I used ngRoute event because I'm not using ngRoute anyway.. but it might cause issues on some cases.

**Comment by p1lare** [comment]
Thanks @taasver !
The HACK, worked for me as well.  Waiting for next version of ngMeta, with the real fix.
Now I hope google will index all my custom meta tags :)
I have some issues or feature request, but I will open them in different topic.

**Comment by Remco75** [comment]
Any plans on this one? Can I help out?

**Comment by so-eugene** [comment]
Any plans???

**Comment by rodrigoyoshida** [comment]
@taasver where can I find the app.run file to add that code?

**Comment by taasver** [comment]
Sorry, there might not be a file called app.run on your case.
But it needs to be in run block.
Maybe this will be help: https://stackoverflow.com/questions/20663076/angularjs-app-run-documentation

**Comment by rodrigoyoshida** [comment]
@taasver thanks! It worked perfectly. Hope they fix the module so we do not have to use this hack.

**Comment by vinaygopinath** [comment]
Should be fixed in [v1.0.2](https://github.com/vinaygopinath/ngMeta/releases/tag/v1.0.2). Please reopen if you still see this issue.
